cleanup.sh

```diff
@@ -3,4 +3,5 @@
 rm -rf /dev/shm/*
 rm -rf /tmp/daos*log
 
-mv Log/$SLURM_JOB_ID $LOGS/log_$1
+NUM_SERVERS=$(printf "%03d" $1)
```

Comment:
This is very rigid. It will break if we decide to go full scale on Frontera i.e. > 1k nodes

Reply:
I would change it to %04d to accommodate up to 9999 servers. The reason to have a fixed width string is based on the use of this parameter.